client/web/web.go

```diff
@@ -74,7 +87,7 @@ type ServerOpts struct {
 }
 
 // NewServer constructs a new Tailscale web client server.
-func NewServer(opts ServerOpts) (s *Server, cleanup func()) {
+func NewServer(ctx context.Context, opts ServerOpts) (s *Server, cleanup func()) {
```

Comment:
document the lifetime/use of the context. Is it only for the duration of the NewServer call, or does is it for the lifetime of the server?

Reply:
Should live for the lifetime of the server, will add a doc